synapse/tools/easycert.py

```diff
@@ -71,6 +72,10 @@ def main(argv, outp=None):
             cdir.genHostCert(opts.name, signas=opts.signas, outp=outp, sans=opts.server_sans)
             return 0
 
+        if opts.client:
+            cdir.genClientCert(opts.name, signas=opts.signas, outp=outp)
```

Comment:
See above note about `signas`. Should I add another flag called something like `caname`?

I wanted to avoid adding a redundant flag, but I don't think it makes sense to rename or use the existing flag.